dask/dataframe/tests/test_groupby.py

```diff
@@ -2329,3 +2329,32 @@ def test_groupby_split_out_multiindex(split_out, column):
     )
 
     assert_eq(ddf_result, ddf_result_so1, check_index=False)
+
+
+def test_groupby_large_ints_exception_cudf():
```

Comment:
This would be the first cudf test within the Dask test suite.  Do we want to have cudf tests in Dask itself, or should these go into the dask-cudf library within the cudf repository?

Reply:
There's another test just above it, `test_groupby_dropna_cdf` which I copied for my test, so it isn't the first time.

Reply:
Ideally we'd have a single test that's executed against different backends. Something like

```python
pytest.mark.parametrize("backend", ["pandas", "cudf"])
def test_groupby_var_large_ints(backend):
    pd = pytest.importorskip(backend)
    series = pd.Series(...)
    ...

```

Reply:
That's tricky only because there are two `cudf` related imports, `cudf` and `dask_cudf`. I can write it with an `if` statement but it won't be as clean as the example you gave. What do you think? Two tests, or one test with an `if` in it?

Reply:
One test, with the if condition I think.

Reply:
Done, look good to merge?